test/browser-only/__snapshots__/constructLayoutEngine-browser.test.js.snap

```diff
@@ -392,17 +392,27 @@ exports[`constructLayoutEngine browser error handling works with error handlers
 <body>
   <div
     id="single-spa-application:header"
-  />
+  >
+    <div>
+      Oops! An error occurred!
+    </div>
+  </div>
   <div
     class="main-content"
   >
     <div
       id="single-spa-application:app1"
     >
-      <div>
-        Oops! An error occurred!
-      </div>
+      error parcel mounted
     </div>
   </div>
 </body>
 `;
+
+exports[`constructLayoutEngine browser error handling works with error handlers defined in HTML 2`] = `
+<body>
```

Comment:
Note that in this snapshot, that both the error parcel and the container div for app1 are gone. This was not the case until I implemented the fix above.